### ISSUE TITLE
The README's Slack url should be permanent (it currently expires every 30 days)

### ISSUE THREAD

**Issue by SnirShechter** [mini-services/miniurl#56]

**Comment by shay123g** [comment]
Hi @SnirShechter 

I would like to be assigne to this

**Comment by SnirShechter** [comment]
@shay123g 
Sounds great.

It's not an easy task, though.

You're free to plan it yourself, however, here's what I had in mind when opening it:
* In case there's a way to create a permanent link, that's great. Just use that.
* In case there isn't, there are two ways to go:
    * We can use a scheduled Github action to run every month and renew the link. The new link can be replaced in the docs etc.
    * We can create a permanent link of our own (given the current temporary link https://miniservices.snir.sh, we can use https://slack.miniservices.snir.sh). That link will be routed to our own server or lambda function which will request an invitation link and redirect to there.
    
What do you think?

**Comment by shay123g** [comment]
@SnirShechter 
there isnt option in Slack to generate permanent invitation link, however:
- there is a way we can try to create permanent link using the Slack worspace management. you can see it here: 
  https://www.reddit.com/r/Slack/comments/e26mn8/automating_slack_invites
  I would like to try that. any chance to get permission to Slack admin page?
- if the first way will not work, I think working with Github actions will be better. only thing we need to do afterwards is to create 
  a script that replace the url in the file

**Comment by SnirShechter** [comment]
@shay123g 

Granted.

**Comment by shay123g** [comment]
> @shay123g
> 
> Granted.
cred? (pm me)
![image](https://user-images.githubusercontent.com/49873692/107222730-f6ac1280-6a1d-11eb-9c43-62115aa69e42.png)

**Comment by SnirShechter** [comment]
I sent you an invitation on email.

However, seems like I solved the issue. Since you didn't manage to login, I created an invitation link and followed the instructions in the link you posted.

This link is set to never expire:
https://join.slack.com/t/mini-services/shared_invite/zt-lu0mqw8n-HVk1Aq6yUODvJXZ6ikIA9w


Issue is solved. You want to do the honors and create the PR to replace the link? 🙂
